source/styles.brs

```diff
@@ -1,78 +1,17 @@
-function GetStyles() as Object
-    return {
-        classes: {
-            ' HomeScene
-            homeList: {
-                itemComponentName: "HomeListItem",
-                numRows:"3",
-                translation: "[50, 150]",
-                itemSize: "[ 2000, 350 ]",
-                showRowLabel: "[ true ]",
-                rowLabelOffset: "[ [0, 15], [0, 15], [0, 15] ]",
-                drawFocusFeedback:true,
-                vertFocusAnimationStyle:"fixedFocusWrap",
-                rowFocusAnimationStyle: "fixedFocusWrap",
-                visible: true,
-                rowItemSize:"[[500, 285]]",
-                itemSpacing:"[0, 80]",
-                rowItemSpacing: " [[30, 0]]"
-            },
-            video: {
-                   width:"1920",
-                   height:"1080",
-                   visible:false
-            },
+function getClassesFromJson () as object
+      info = ReadAsciiFile("pkg:/source/styles.json")
+      json = ParseJson(info)
 
-            videoDetail:{
-                visible: false
-            },
+      'contentNode = CreateObject("roSGNode", "ContentNode")
```

Comment:
leaving commented code is not a good practice (unless you have a very good reason)

Reply:
Okay, thanks! I left it in case I would have to reuse it.

Reply:
Thank you for the notice.

Reply:
In my opinion, if there's any reason to left commented code, there should be also a clear comment about its meaning, so, everyone can understand the propose of that